mkdocs_git_revision_date_localized_plugin/util.py

```diff
@@ -117,6 +120,7 @@ def get_git_commit_timestamp(
             realpath = os.path.realpath(path)
             git = self._get_repo(realpath)
 
+            # Ignored commits are only considered for the most recent update, not for creation
```

Comment:
With 150k downloads per month, someone is bound to stumble on this. It should ignore those commits everywhere.

Best to refactor, using a separate function that takes in a list of commits and return the last or first one that is not in the ignored commits list. You can use that function for both creation and most recent update.

We also need a proper fallback inside that function. What if all commits are ignored (f.e. because there is only 1)? You would think that never happens, but it could happen for new repos. Or, some users, f.e. when using CI, use a shallow clone, that only retrieves the last commit. If that commits happens to be an ignored commit -- there are no more commits left to determine the build date.

Probably the best fallback for that scenario is to then use ignored commits anyway. We'd need to document that. And we probably want to inform the user that that happened. It should log a warning when the `strict`  option is `True`.